Makefile

```diff
@@ -13,6 +13,8 @@ export
 #============================================================================
 
 all:			## Build and launch all services
+	@printf "\033[1;33mWARNING:\033[0m You may take a while due to the enormous size of the Triton server image, and this image pulling process is a one-time effort.\n"
```

Comment:
```suggestion
	@printf "\033[1;33mWARNING:\033[0m You may take a while due to the enormous size of the Triton server image, but the image pulling process should be just a one-time effort.\n"
```

Reply:
Let's also add a check for the existence of images before sending out the warning, instead of warning all the time.